src/pages/components/content-switcher/code.mdx

```diff
@@ -4,34 +4,180 @@ description: Content switchers manipulate the content shown following an exclusi
 tabs: ["Usage", "Style", "Code", "Accessibility"]
 ---
 
-<ComponentDemo
-  components={[
-    {
-      id: "content-switcher",
-      label: "Content switcher",
-    },
-  ]}
->
-  <ComponentVariant
-    id="content-switcher"
-    knobs={{
-      Switch: ["disabled"],
-    }}
-    links={{
-      React:
-        "http://react.carbondesignsystem.com/?path=/story/contentswitcher--default",
-      Angular:
-        "https://angular.carbondesignsystem.com/?path=/story/components-content-switcher--basic",
-      Vue:
-        "http://vue.carbondesignsystem.com/?path=/story/components-cvcontentswitcher--default",
-      Vanilla:
-        "https://the-carbon-components.netlify.com/?nav=content-switcher",
-    }}
-  >{`
-<ContentSwitcher onChange={console.log}>
-  <Switch name={'first'} text='First section' />
-  <Switch name={'second'} text='Second section' />
-  <Switch name={'third'} text='Third section' />
+<PageDescription>
+
+How to build a content switcher using React. For code usage with other frameworks, please follow the links in the [live demo](https://www.carbondesignsystem.com/components/content-switcher/usage#live-demo).
```

Comment:
```suggestion
How to build a content switcher using React. For code usage with other frameworks, please follow the links in the live demo on the usage tab.
```

Reply:
oh yes good call i forgot to update that thanks